app/models/vm_or_template/operations/relocation.rb

```diff
@@ -107,6 +107,7 @@ def raw_move_into_folder(folder)
   end
 
   def move_into_folder(folder)
+    folder = EmsFolder.find_by(:ems_ref => folder) if folder.kind_of?(String)
```

Comment:
ems_ref is insufficient to guarantee uniqueness globally, you need either ems_id + ems_ref or id